external/external-commons/src/main/java/edu/illinois/cs/cogcomp/pipeline/main/ExternalAnnotatorServiceFactory.java

```diff
@@ -29,8 +29,10 @@
      * @throws IOException
      * @throws AnnotatorException
      */
-    public static BasicAnnotatorService buildPipeline(Map<String, Annotator> annotators) throws IOException, AnnotatorException {
-        ResourceManager emptyConfig = new ResourceManager(new Properties());
+    public static BasicAnnotatorService buildPipeline(Map<String, Annotator> annotators, String cacheName) throws IOException, AnnotatorException {
```

Comment:
Why change the existing factory method, rather than adding another? we already have many factory methods, so I understand the desire to avoid creating additional ones; but if this parameter is so important, shouldn't other constructors also be modified?

nitpick: choose a name like "cacheDirectory" or "cachePath" rather than "cacheName"

Reply:
- parameter name: changed. 
- changing the factory name: slightly confused about your comment: are you suggesting adding a another method? Or are you suggesting adding the same parameter to the other constructor? (the answer to which is, the other constructor accepts s ResourceManager parameter, which contains this param).